views/overlays/commentators.hbs

```diff
@@ -9,75 +9,227 @@
+                <path d="M135.16,202.260233 L90.664,319.541049 L87.496,320.611916 L0.72820282,149.869174" id="V" class="bumper-svg__letter" stroke="#FFFFFF" stroke-width="4" stroke-dashoffset="1000" stroke-dasharray="1000"></path>
+              </g>
+            </g>
+          </g>
+        </svg>
+      </div>
+    </div>
     <script>
     var processedFollowers = [];
     var socket = io('/overlay');
 
     //TODO global
-    var transitioning = false;
-    function transition() {
-      if(!transitioning) {
-        transitioning = true;
-         //Slide in first
-         $('#transition-image').effect('slide', {}, 1000, function() {
-          //Wait 2 seconds
-          setTimeout(function() {
-            //Blind out of view
-            $('#transition-image').effect('fold', {}, 1000, function() {
-              transitioning=false;
-            });
-          }, 2500);           
-         });
-      }
-    }
-
+    
+    
     socket.on('update overlay', function(data) {
-        if(data.commentators != document.getElementById('commentators').innerText) {
-          $('#commentators').trigger('textChanged', data.commentators || '');
+        var shouldReanimate = false;
+        if(data.commentators != $('.commentators-info__text--left-top').text().trim()) {
+          $('.commentators-info__text--left-top').trigger('textChanged', data.commentators || '');
+          shouldReanimate = true;
+        }
+        if(data.commentators2 != $('.commentators-info__text--right-top').text().trim()) {
+          $('.commentators-info__text--right-top').trigger('textChanged', data.commentators2 || '');
+          shouldReanimate = true;
         }
-        if(data.commentators2 != document.getElementById('commentators2').innerText) {
-          $('#commentators2').trigger('textChanged', data.commentators2 || '');
+        if(data.twitter != $('.commentators-info__text--left-bottom').text().trim()) {
+          $('.commentators-info__text--left-bottom').trigger('textChanged', data.twitter || '');
+          shouldReanimate = true;
         }
-        if(data.twitter != document.getElementById('twitter').innerText) {
-          $('#twitter').trigger('textChanged', data.twitter || '');
+        if(data.twitter2 != $('.commentators-info__text--right-bottom').text().trim()) {
+          $('.commentators-info__text--right-bottom').trigger('textChanged', data.twitter2 || '');
+          shouldReanimate = true;
         }
-        if(data.twitter2 != document.getElementById('twitter2').innerText) {
-          $('#twitter2').trigger('textChanged', data.twitter2 || '');
+        
+        if(shouldReanimate) {
+            animateCommentatorNamesIn();
+            setTimeout(animateCommentatorNamesOut, 30000);
```

Comment:
For this one, no. I tested that, and it just updates the names immediately and hides based on the original invocation (since we're just adding and removing classes).